src/main/resources/templates/fragments/accordion.html

```diff
@@ -8,7 +8,7 @@
     <button class="accordion__button" aria-expanded="true" aria-controls="controlId">
       <th:block th:replace="${buttonLabel}"/>
     </button>
-    <div class="accordion__content" id="controlId">
+    <div class="accordion__content" aria-expanded="false" id="controlId">
```

Comment:
This needs to get updated by JavaScript (which I believe Honeycrisp does automatically?) or it won't work with screen readers. 

Let me know if you wanna pair on testing it :)